src/Chapter2.hs

```diff
@@ -620,7 +624,14 @@ Write a function that takes elements of a list only on even positions.
 >>> takeEven [2, 1, 3, 5, 4]
 [2,3,4]
 -}
-takeEven = error "takeEven: Not implemented!"
+takeEven :: [a] -> [a]
+takeEven list = selector 0 [] list
+  where
+    selector :: Int -> [a] -> [a] -> [a]
+    selector index evenList [] = evenList
+    selector index evenList (x:xs) = if (mod index 2 == 0) 
+                              then selector (index + 1) (evenList ++ [x]) xs
+                              else selector (index + 1) evenList xs
```

Comment:
Creative again! 💡 

You actually could implement this one with just pattern matching. You would need to check a few cases of the list for that 😉